### PR TITLE
PIM-5119: Manage filters very slow when a lot of attributes to display

### PR DESCRIPTION
**Pull request by grena**
| Q                 | A
| ----------------- | ---
| Specs             | Y
| Behats            | Y
| Blue CI           | Y
| Changelog updated | Y
| Review and 2 GTM  | Y

### CODE REVIEW

**Comment by jjanvier** [comment]
add the ticket number to your second commit

**Comment by jjanvier** [comment]
GTM except my comment (and no troll, I really understood the fix as I saw how adrien did it)

File: src/Pim/Bundle/DataGridBundle/Resources/public/js/multiselect-decorator.js [hunk 1]
```diff
@@ -37,6 +37,13 @@ function($, _, mediator) {
         contextSearch: true,
 
         /**
+         * Minimum width of this multiselect
+         *
+         * @property {int}
+         */
+        minimumWidth: null,
```
Comment (sumbobyboys) [inline src/Pim/Bundle/DataGridBundle/Resources/public/js/multiselect-decorator.js]: CSS in javascript?

Comment (sumbobyboys) [inline src/Pim/Bundle/DataGridBundle/Resources/public/js/multiselect-decorator.js]: ok, old stuff, sorry

File: src/Pim/Bundle/DataGridBundle/Resources/public/js/multiselect-decorator.js [hunk 2]
```diff
@@ -114,32 +121,21 @@ function($, _, mediator) {
...
-            var elements = this.getWidget().find('.ui-multiselect-checkboxes li');
-            _.each(elements, function(element) {
-                var width = this._getTextWidth($(element).find('label'));
-                if (width > minimumWidth) {
-                    minimumWidth = width;
-                }
-            }, this);
+            if (_.isNull(this.minimumWidth)) {
+                var elements = this.getWidget().find('.ui-multiselect-checkboxes li');
+                var margin = 26;
```
Comment (sumbobyboys) [inline src/Pim/Bundle/DataGridBundle/Resources/public/js/multiselect-decorator.js]: !()[http://33.media.tumblr.com/tumblr_m9knmh4Wa51rdelzeo1_500.gif]

**Comment by solivier** [comment]
GTM

**Comment by sumbobyboys** [comment]
GTM

**Pull request merged**